dev.yml

```diff
@@ -0,0 +1,39 @@
+name: identity_cache
+
+up:
+  - homebrew:
+    - postgresql
+  - ruby:
+      version: 2.2.3p172-shopify
+      package: shopify/shopify/shopify-ruby
+  - railgun
+  - bundler
+
+commands:
+  test:
+    syntax:
+      argument: file
+      optional: args...
+    run: |
+      export RAILGUN_HOST=identity_cache.railgun
```

Comment:
Instead of this you can add a section to the yaml toplevel:

```yaml
env:
  RAILGUN_HOST: identity_cache.railgun
  MYSQL_HOST: identity_cache.railgun
  POSTGRES_HOST: identity_cache.railgun
  MEMCACHED_HOST: identity_cache.railgun
```